odltools/monitor/monitor_odl_cluster.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/python
+
+# Copyright (c) 2018 Red Hat Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""
+Cluster Monitor Tool
+Original Author: Phillip Shea
```

Comment:
can we just put this in the AUTHORS file

Reply:
No more authors file.  Was just trying to retain some credit to the original author.  I'll remove it though.